hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -96,10 +96,14 @@ def on_update(self):
 		self.publish_update()
 		self.notify_approval_status()
 
+	def before_submit(self):
+		if self.status == "Open":
+			self.status = "Approved"
```

Comment:
Let's not assume this. People might have workflows that would break. We can revert the changes in `leave_application.py`